api/rpc/endpoints.go

```diff
@@ -6,20 +6,24 @@ import (
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-func (s *Server) GetLatestData(ctx context.Context, _ *emptypb.Empty) (*protobuff.GetLatestDataResponse, error) {
+func (s *Server) GetLatestData(_ context.Context, _ *emptypb.Empty) (*protobuff.GetLatestDataResponse, error) {
```

Comment:
in 99% of the cases methods should resider near their receiver structs, in the same file